frontend/cypress/e2e/dashboard.cy.js

```diff
@@ -4,82 +4,158 @@ let homePage = null;
 let loginPage = null;
 let dashboard = null;
 
-before("login", () => {
+// Helper function to log in and navigate to the homepage
+const loginAndNavigateToHome = () => {
   loginPage = new LoginPage();
   loginPage.visit();
-});
+  homePage = loginPage.goToHomePage();
+};
+
+// Helper function to add a new order
+const addNewOrder = (dashboardType, testType, sampleType, panelType) => {
+  homePage.goToOrderPage();
```

Comment:
does this function really add a coresponding program ??
i dont see where `dashboardType` is used with in the function

Reply:
ALright , i think i had confused dashboardtype with the program